src/components/table/examples/table-selectable-rows.tsx

```diff
@@ -78,6 +72,16 @@ export class TableExampleSelectableRows {
                 movableColumns={true}
                 class="has-interactive-rows"
             />,
+            <limel-example-controls
+                style={{ '--example-controls-max-columns-width': '10rem' }}
+            >
+                <limel-button label="Select all" onClick={this.selectAll} />
+                <limel-button
+                    label="Clear selection"
+                    onClick={this.clearSelection}
+                />
+                <limel-button label="Reset" onClick={this.resetSelection} />
+            </limel-example-controls>,
```

Comment:
The buttons look worse after this change 😕 Not the end of the world, but shouldn't there at least be some default gutter between rows? 🤔

Before:

<img width="513" alt="image" src="https://user-images.githubusercontent.com/154725/210373580-a6f6b18e-7e9b-4bd1-b03b-096db4a2860b.png">

After:

<img width="513" alt="image" src="https://user-images.githubusercontent.com/154725/210373501-062ca74f-c4a6-42cd-a035-3963e4c1d4af.png">

Reply:
Oh I see. I intentionally made the example-controls' gap to be zero `gap: 0 1rem` just because for many checkboxes or toggle buttons it felt we don't need `row-gap`. They already had enough distance between them. But when there are buttons (or input fields), then we would need row gap. So we should either:
- set the row gap to be very little, like `gap: 0.25rem 1rem;` or 
- make a CSS variable for the row gap

Reply:
Ah, yeah, I guess we might want to polish this a little later on then. But for now, I guess it's better that the checkboxes doesn't get loads of extra space between them, and this example can look a little off. Or maybe I'll do something about it, we'll see… 😄

Reply:
⚡

I set `gap: 0.25rem 1rem;`. It looks nice for both buttons and checkboxes 👍